app/controllers/SomeInformationIsMissingController.scala

```diff
@@ -30,17 +30,15 @@ class SomeInformationIsMissingController @Inject() (
   override val messagesApi: MessagesApi,
   standardActionSets: StandardActionSets,
   val controllerComponents: MessagesControllerComponents,
-  renderer: Renderer
+  view: SomeInformationIsMissingView
 )(implicit ec: ExecutionContext)
     extends FrontendBaseController
     with I18nSupport {
 
   def onPageLoad(): Action[AnyContent] =
-    standardActionSets.identifiedUserWithData().async {
+    standardActionSets.identifiedUserWithData() {
       implicit request =>
-        val data = Json.obj(
-          "continue" -> routes.IndexController.onPageLoad().url
-        )
-        renderer.render("someInformationIsMissing.njk", data).map(Ok(_))
+        val continueUrl = routes.DoYouHaveUniqueTaxPayerReferenceController.onPageLoad(NormalMode).url
```

Comment:
continueUrl should go back to the indexController note that it was "continue" -> routes.IndexController.onPageLoad().url previously.